packages/decent.js/docs/channel.md

```diff
@@ -34,6 +34,12 @@ Resolves with an array (not set!) of [message](message.md)s of max length `limit
 ## await channel.getPins() -> [PinnedMessages](pinned-messages.md)
 Resolves with a PinnedMessages set containing the messages pinned to this channel.
 
+## await channels.rename(newName: string)
+Renames this channel.
+
+## await channels.delete()
+Deletes this channel.
+
```

Comment:
Should these be `channel.rename` and `channel.delete` (instead of <code>channel<strong>s</code></code>)?

Reply:
Yup.

Reply:
Same deal with `channel.create`.